src/components/manage/Actions/Actions.jsx

```diff
@@ -9,12 +9,11 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { Dropdown, Icon } from 'semantic-ui-react';
-import { toast } from 'react-toastify';
 import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
 
 import { cut, copy, copyContent, moveContent } from '@plone/volto/actions';
 import { getBaseUrl } from '@plone/volto/helpers';
-import { ContentsRenameModal, Toast } from '@plone/volto/components';
+import { ContentsRenameModal, Toast, toast } from '@plone/volto/components';
```

Comment:
I propose that we get rid of the `Toast` and only use the `toast()` as API (maybe allow passing the Toast as a custom optional parameter). It seems redundant to do `toast(<Toast>)`

@sneridagh ?

Reply:
I also agree